jaxlib/cuda_lu_pivot.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2021 Google LLC
```

Comment:
This is fine, but I have to wonder if this is the last hand-written linear algebra CUDA kernel we will have to add... I might have been tempted to name the module something like `cuda_linalg.py`, but we can always do that if and when we have a second kernel.